File: samples_generated_code/sample_1/phi4/baseline_code_py/code_row_81.py
import sys

def getValueFromArray(arr, size, index):
    if 0 <= index < size:
        return arr[index]
    else:
        raise IndexError("Index out of bounds")

if __name__ == "__main__":
    id_sequence = [1, 2, 3, 4, 5]  # Example array; adjust as necessary
    try:
        index = int(input().strip())
        value = getValueFromArray(id_sequence, len(id_sequence), index)
        sys.exit(value)
    except (ValueError, IndexError):
        sys.exit(1)

File: samples_generated_code/sample_3/athene/baseline_code_c/code_row_81.c
#include <stdio.h>
#include <stdlib.h>

int getValueFromArray(int *arr, int size, int index) {
    if (index >= 0 && index < size) {
        return arr[index];
    }
    return -1;
}

int main() {
    int id_sequence[] = {1, 2, 3, 4, 5};
    int size = sizeof(id_sequence) / sizeof(id_sequence[0]);
    int index;

    scanf("%d", &index);
    int value = getValueFromArray(id_sequence, size, index);

    return value;
}